src/keys/verification_scheduler_key.json

```diff
@@ -1,270 +1,270 @@
 {
```

Comment:
but changing this means, that the previous proofs will stop working, right?

Reply:
updated.